app/views/state_file/questions/nj_disabled_exemption/edit.html.erb

```diff
@@ -0,0 +1,41 @@
+<%
+    title = t(".title", filing_year: @filing_year)
+    content_for :page_title, title
+%>
+
+<% content_for :card do %>
+  <h1 class="h2"><%= t(".title") %></h1>
+  <p class="p"><%= t(".instructions_html") %></p>
+  <%= form_with model: @form, url: { action: :update }, local: true, method: "put", builder: VitaMinFormBuilder do |f| %>
+    <% if !current_intake.direct_file_data.is_primary_blind?%>
+    <div class="white-group">
+      <p class="p"><%= t(".primary_disabled_question") %></p>
+      <%= f.cfa_radio_set(:primary_disabled, 
```

Comment:
[pebble] Instead of using a p tag that isn't programmatically associated at all, and then a sr-only legend, we should just make the text that's currently in the p tag the legend and remove the sr-only legend class

Reply:
Thanks! Wasn't familiar with best practices around legends, sr-only tags, etc. Appreciate you sharing your knowledge.